<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExaminationForm</class>
 <widget class="QWidget" name="ExaminationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>ImiÄ™</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Data i czas</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateDateTimeEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="descLabel">
      <property name="text">
       <string>Opis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="descLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="scanNameLabel">
      <property name="text">
       <string>Plik scanu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="scanNameLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
